src/components/ExpandedNode/ExpandedNode.tsx

```diff
@@ -154,7 +187,7 @@ function renderBranch(
       <ExpandedNodeField
         key="value"
         title="Value"
-        description={<MissingDataPopup values={values} />}
+        description={<MissingDataPopup values={values.filter((a, b) => values.indexOf(a) === b)} />}
```

Comment:
Nitpick but it feels cleaner to do this "unique" test up above where the `values` field is first set. Also could we add a comment explaining that this is a unique test, it's not immediately obvious that's what it's doing.

Reply:
Addressed  this.